src/cloudformation_cli_python_lib/interface.py

```diff
@@ -90,9 +90,21 @@ def _serialize(
         # mutate to what's expected in the response
         if to_response:
             ser["bearerToken"] = bearer_token
-            ser["operationStatus"] = ser.pop("status")
-            if ser["callbackDelaySeconds"] == 0:
-                del ser["callbackDelaySeconds"]
+            ser["operationStatus"] = ser.pop("status").name
+            if self.resourceModel:
+                # pylint: disable=protected-access
+                ser["resourceModel"] = self.resourceModel._serialize()
+            if self.resourceModels:
+                ser["resourceModels"] = [
+                    # pylint: disable=protected-access
+                    model._serialize()
+                    for model in self.resourceModels
+                ]
+            del ser["callbackDelaySeconds"]
+            if "callbackContext" in ser:
+                del ser["callbackContext"]
+            if self.errorCode:
+                ser["errorCode"] = self.errorCode.name
```

Comment:
fine for now, i think in future we'll want to have another class to be the cloudformation callback struct and have a method to convert from one to the other